src/main/java/paymelah/model/Model.java

```diff
@@ -84,4 +85,10 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Sorts the address book's person list by the given {@code comparator}.
```

Comment:
Should there be a @param as well?

Reply:
I don't think need to have complete JavaDocs for simple methods like this. The updateFilteredPersonList also did not have a @param